config/sidekiq_scheduler.yml

```diff
@@ -149,3 +149,9 @@ VaForms::FetchLatest:
   cron: "0 0 * * * America/New_York"
   class: VaForms::FormReloader
   description: "Fetches latest VA Forms"
+
+ExportGithubStats:
+  cron: "0 */2 * * * America/New_York"
+  class: GithubStatsJob
+  description: "Send Github open PR stats to prometheus"
+  enabled: <%= Settings.hostname == 'dev-api.vets.gov' %>
```

Comment:
I thought we were going with a Jenkins job to run the rake task instead a sidekiq scheduler job - what gives? (I'm not necessarily opposed to the scheduler strategy, I just didn't expect to see it)

Reply:
Good catch. I totally forgot to remove that.